internal/receiver/scriptreceiver/README.md

```diff
@@ -0,0 +1,84 @@
+| sshdChecker	        | Parses sshd_config for information local sshd configurations                                                                                                                                                                |
+| time	                | System date and time, and NTP server time via the shell commands and chronyc, date andntpdate                                                                                                                               |
+| top	                | List of running system processes via the shell commands ps and top                                                                                                                                                          |
+| update               | 	Available software updates for installed packages via the shell commands softwareupdate and yum                                                                                                                           |
+| uptime               | 	System date and uptime information via the shell command date                                                                                                                                                             |
+| usersWithLoginPrivs  | 	Shows system username information                                                                                                                                                                                         |
+| version	            | OS version details via the shell command uname                                                                                                                                                                              |
+| vmstat	            | Process-related memory usage information via the shell commands prstat, prtconf, ps, sar, svmon, swap, swapinfo, sysctl, top, uptime, and vmstat                                                                            |
+| vsftpdChecker        | 	Parses vsftpd.conf for information about local VSFTP server configurations in /etc, /etc/vsftpd, or /private/etc                                                                                                          |
+| who	                | Information about all users currently logged in via the shell command who                                                                                                                                                   |
+
+The following settings are required:
+
+- `script_name` : Name of the script to be executed.
+- `interval` : (default = `60s`) how often the script should be executed
+
+
+The following settings are optional:
+
+- `source` : source of the event
+- `sourcetype` : sourcetype of the event
+- `multiline` : how the standard output of the script is split
```

Comment:
any chance you can detail the supported values/sub-config or is this still in flux?